infra/storage/spanner/migrations/000012.sql

```diff
@@ -0,0 +1,38 @@
+-- Copyright 2024 Google LLC
+--
+-- Licensed under the Apache License, Version 2.0 (the "License");
+-- you may not use this file except in compliance with the License.
+-- You may obtain a copy of the License at
+--
+--     http://www.apache.org/licenses/LICENSE-2.0
+--
+-- Unless required by applicable law or agreed to in writing, software
+-- distributed under the License is distributed on an "AS IS" BASIS,
+-- WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+-- See the License for the specific language governing permissions and
+-- limitations under the License.
+
+-- This index is designed to optimize queries that need to quickly identify the support status of
+-- a specific feature across different browsers and releases. This is particularly useful for
+-- identifying "missing one implementation" scenarios, where a feature is supported by all
+-- browsers except the target browser on a given release date.
+-- The descending order on EventReleaseDate supports efficient retrieval
+-- of the most recent releases.
+CREATE INDEX BrowserFeatureSupportEvents_FeatureStatusOnBrowserRelease ON
+    BrowserFeatureSupportEvents(
```

Comment:
Minor details: how often `BrowserFeatureSupportEvents` is updated? Having indexes does slightly impact the write performance of the table

Reply:
Good question! Twice a day.

https://github.com/GoogleChrome/webstatus.dev/blob/31c79b507f4de108d08ce60c63f9f3771d459732/infra/.envs/prod.tfvars#L57-L60

I did deploy the index sometime last week to test this out. The jobs are taking around the same time before and after.
- [us-central](https://pantheon.corp.google.com/run/jobs/details/us-central1/staging-us-central1-web-features/executions?project=webstatus-dev-internal-staging)
- [europe](https://pantheon.corp.google.com/run/jobs/details/europe-west1/staging-europe-west1-web-features/executions?project=webstatus-dev-internal-staging)

(on an unrelated note, you will see that the Europe job is faster. That is because the [database](https://pantheon.corp.google.com/spanner/instances?referrer=search&project=webstatus-dev-internal-staging) itself is in europe in staging).